tests/src/Interop/PInvoke/Primitives/Pointer/PInvokePointerTest.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <DefineConstants>$(DefineConstants);STATIC</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="PInvokePointerTest.cs" />
+  </ItemGroup>
+  <Import Project="../../../Interop.settings.targets" />
```

Comment:
Please change this to `<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Interop.settings.targets))\Interop.settings.targets" />`

Reply:
Yup, will do. Note that I copy-pasted the pattern from existing files under `tests/src/PInvoke`.